server/index.js

```diff
@@ -0,0 +1,29 @@
+// Starting up the server
+// Use the require method on express
+// Create variable to hold it
+const express = require('express');
+
+// Create an instance of an Express application
+const app = express(); // => A function provided by the Express.js framework
+// App object represents web application
+// Use app object to define routes middleware and other functionalities
+
+// Setting up middleware
+app.use(express.json()); // Parse the request body
+app.use(express.urlencoded()); // Parses url
+app.use(express.static()); // Statically serve up client directory
```

Comment:
the argument in express.static() needs to be a path to our /dist directory. In the root directory, run `node server/index.js` and you'll see an error pop up

Reply:
also: app.use(express.urlencoded({extended: true})